fcrepo-auth-webac/src/test/java/org/fcrepo/auth/webac/WebACFilterTest.java

```diff
@@ -174,10 +174,11 @@ public void setupRequest() throws Exception {
 
         when(mockTransactionManager.get(transactionId)).thenReturn(mockTransaction);
 
-        when(mockResourceFactory.getResource(mockTransaction, testChildId))
+        when(mockResourceFactory.getResource(mockTransaction.getId(), testChildId))
```

Comment:
Seems like this and all the other instances modified in this class could be reverted back to just passing in the `mockTransaction`